### DISCUSSION TITLE
markdown

### DISCUSSION THREAD

**Discussion by maidey** [CaiJimmy/hugo-theme-stack#389] [Q&A]
How to resize .svg in markdown?

**Comment by QIN2DIM** [comment]
```markdown
<div align="center"><img src="your.svg" alt="caption" style="zoom:80%; object-fit: cover;"/></div>
```

**Comment by maidey** [comment]
I know, 
But my question is markdown, not HTML

**Comment by CaiJimmy** [comment]
It ain't part of the Markdown standard. As far as I know there's no way to do that using pure Markdown syntax.

BTW this is not a theme issue, use Discussion instead, please.
